examples/seismic/stiffness/utils.py

```diff
@@ -0,0 +1,78 @@
+from devito.types.tensor import tens_func
+import numpy as np
+
+
+def D(self, shift=None):
+    """
+    Returns the result of matrix D applied over the TensorFunction.
```

Comment:
Would it be worth showing what matrix D contains or pointing to a paper using this notation in the docstring?

Reply:
Maybe just repeat the Feng and Schuster (2017): Elastic least-squares reverse time migration https://doi.org/10.1190/geo2016-0254.1 here?